src/pages/resultsView/survival/SurvivalChartExtended.tsx

```diff
@@ -0,0 +1,1684 @@
+                </div>
+            </div>
+        );
+    }
+
+    @computed get chartTooltip() {
+        return (
+            <div style={{ maxWidth: 250 }}>
+                {this.analysisGroupsWithData.map(group => (
+                    <span
+                        key={group.value}
+                        style={{
+                            display: 'inline-block',
+                            marginRight: 12,
+                            marginBottom: 20,
+                            fontWeight:
+                                this.highlightedCurve === group.value
+                                    ? 'bold'
+                                    : 'initial',
+                            cursor: 'pointer',
+                        }}
+                        onClick={() => {
+                            this.highlightedCurve =
+                                this.highlightedCurve === group.value
+                                    ? ''
+                                    : group.value;
+                        }}
+                    >
+                        <div
+                            style={{
+                                width: 10,
+                                height: 10,
+                                display: 'inline-block',
+                                backgroundColor: group.color,
+                                marginRight: 5,
+                            }}
+                        />
+                        {!!group.name ? group.name : group.value}
+                    </span>
+                ))}
+                {this.props.showNaPatientsHiddenToggle && (
+                    <div className="checkbox">
+                        <label>
+                            <input
+                                type="checkbox"
+                                checked={this.props.naPatientsHiddenInSurvival}
+                                onClick={
+                                    this.props.toggleSurvivalHideNAPatients
+                                }
+                            />{' '}
+                            Exclude patients with NA for any of the selected
+                            attributes.
+                        </label>
+                    </div>
+                )}
+            </div>
+        );
+    }
+
+    @computed get tableRows() {
+        return this.props.analysisGroups.map(grp => (
+            <tr>
+                <td>{!!grp.name ? grp.name : grp.value}</td>
+                {getStats(
+                    this.props.sortedGroupedSurvivals[grp.value],
+                    this.survivalSummaries[grp.value]
+                ).map(stat => (
+                    <td>
+                        <b>{stat}</b>
+                    </td>
+                ))}
+            </tr>
+        ));
+    }
+
+    @computed get tableRowsHazardRatio() {
+        if (this.hazardRatioGroups !== null) {
+            return (
+                <React.Fragment>
```

Comment:
i don't think you need Fragment here since you are just returning a single element (table)